lib/customer.rb

```diff
@@ -0,0 +1,37 @@
+require "csv"
+require_relative "../lib/customer.rb"
+
+class Customer
+  include Comparable
+
+  attr_reader :id
+  attr_accessor :email, :address
+
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+
+  def self.all
+    @customers = []
+    CSV.open("../data/customers.csv").each do |row|
+      id = row[0].to_i
```

Comment:
This path has an extra `../` at the beginning, which made some of the tests fail for me. I suspect that means you're `cd`ing into the `lib` or `specs` directories to run your files. Instead, you should run everything from the project root (the top-level folder that contains `lib/` and `specs/`).